setup.sh

```diff
@@ -61,11 +63,21 @@ function _preparation() {
   fi
   apt-get -q -y update >> ${log} 2>&1
   apt-get -q -y upgrade >> ${log} 2>&1
-  apt-get -q -y install whiptail git sudo curl wget lsof fail2ban apache2-utils vnstat tcl tcl-dev build-essential dirmngr apt-transport-https python-pip >> ${log} 2>&1
+
+  if [[ $codename =~ ("focal") ]]; then 
+      pipv='python3-pip'
+      else
+      pipv='python-pip'
+  fi
```

Comment:
pip is not a direct dependency of swizzin, thus it does not need to be installed here.